Source/WebCore/page/CaptionUserPreferencesMediaAF.cpp

```diff
@@ -461,6 +462,18 @@ String CaptionUserPreferencesMediaAF::captionsDefaultFontCSS() const
     if (!name)
         return emptyString();
 
+    if (fontNameIsSystemFont(name.get())) {
+        if (CFStringHasPrefix(CFSTR(".AppleSystemUIFontMonospaced"), name.get()))
+            name = CFSTR("system-ui-monospaced");
+        else if (CFStringHasPrefix(CFSTR(".AppleSystemUIFont"), name.get()))
```

Comment:
The `else if` and `else` produce the same result.
Do we need to keep the `else if`?

Reply:
I'm setting us up to add more `else if` cases without a bunch of re-writing existing conditions.